custom_components/remote_homeassistant/manifest.json

```
@@ -1,18 +1,19 @@
 {
   "domain": "remote_homeassistant",
   "name": "Remote Home-Assistant",
-  "issue_tracker": "https://github.com/custom-components/remote_homeassistant/issues",
-  "documentation": "https://github.com/custom-components/remote_homeassistant",
-  "dependencies": ["http"],
-  "config_flow": true,
+  "version": "2024.06.18",
```

```suggestion
  "version": "4.0",
```

not sure we should switch to date-based versioning here? for some reason, custom components don't seem to do that usually (perhaps to not establish a false correlation with potential HA releases)

---

This was my personal component until today...  No problem :-)

---

I think you did not click the "commit changes" button here? Would be great if this can still be updated before the release is created (you should be able to change it directly in the main branch)